app/scripts/collections/search_results.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think we discussed this earlier, but it's mildly weird to have underscores in filenames in _/app/*_ but dashes in _/server/*_.

For example: _/app/scripts/collections/**search_results.js**_ vs _/server/work-queue/jobs/**extract-page.js**_.